File: turtlecontrol/turtlecontrol/twistControl.py
#!/usr/bin/env python3
import rclpy
from rclpy.node import Node
from example_interfaces.msg import String
from geometry_msgs.msg import Twist


class TurtleTwistPub(Node): # MODIFY NAME
    def __init__(self):
        super().__init__("turtle_twist") # MODIFY NAME
        self.publisher_= self.create_publisher(Twist, "/turtle1/cmd_vel",10)
        self.timer_ =self.create_timer(0.5,self.publish_news)
        self.get_logger().info("velocity publisher started")
    def publish_news(self):
        msg = Twist()
        msg.linear.x = -2.0
        msg.linear.y = -2.0
        msg.linear.z =0.0
        msg.angular.x = 0.0
        msg.angular.y = 2.0
        msg.angular.z = 1.0
        self.publisher_.publish(msg)
       

def main(args=None):
    rclpy.init(args=args)
    node = TurtleTwistPub() # MODIFY NAME
    rclpy.spin(node)
    rclpy.shutdown()


if __name__ == "__main__":
    main()

File: turtlecontrolcpp/src/twistControl.cpp
#include "rclcpp/rclcpp.hpp"
#include "example_interfaces/msg/string.hpp"
#include "geometry_msgs/msg/twist.hpp"
class TwistControlNode : public rclcpp::Node
{
public:
   TwistControlNode() : Node("robot_news_station")
    {
        publisher_ = this->create_publisher<geometry_msgs::msg::Twist>("turtle1/cmd_vel", 10);
        timer_ = this->create_wall_timer(std::chrono::milliseconds(50),
                                         std::bind(&TwistControlNode::publishNews, this));
        RCLCPP_INFO(this->get_logger(), "Sending Twist instructions.");
    }

private:
    void publishNews()
    {
        auto msg = geometry_msgs::msg::Twist();
        msg.linear.x = 0.5;
        msg.angular.z =0.5;
        publisher_->publish(msg);
    }


    rclcpp::Publisher<geometry_msgs::msg::Twist>::SharedPtr publisher_;
    rclcpp::TimerBase::SharedPtr timer_;
};

int main(int argc, char **argv)
{
    rclcpp::init(argc, argv);
    auto node = std::make_shared<TwistControlNode>();
    rclcpp::spin(node);
    rclcpp::shutdown();
    return 0;
}
